plutus-tx-plugin/test/StdLib/Spec.hs

```diff
@@ -100,28 +97,6 @@ testQuotRem = Hedgehog.property $ do
     Hedgehog.annotateShow plutusResult
     Hedgehog.assert (ghcResult == plutusResult)
 
-testReduce :: Property
```

Comment:
was this test important?

Reply:
It was important, but not anymore:
This test was checking that `PlutusTx.Ratio.reduce` returns the same results as `GHC.Real.reduce` for the same arguments. However, after removing the `PlutusTx.Ratio.reduce` there is nothing left to test, so I dropped the test.